src/util/formatTechRecord.ts

```diff
@@ -0,0 +1,55 @@
+const buildArray = (techRecordWithoutArrays: object, arrayName: string, formattedTechRecord: any): object => {
```

Comment:
The es-lint ignore will improve once we have correct types, so let's ignore for now as it will be fixed later :)